accelerators-home/modules/ROOT/pages/simplifying-assets.adoc

```diff
@@ -0,0 +1,70 @@
+
+* Follow the API-led connectivity approach but create APIs only where necessary to support clean integration with external apps and systems.
+* Keep exception handling clean and simple, allowing for straightforward extension and customization. 
+* Avoid the use of heavy logging, monitoring, or exception handling frameworks.
+* Leverage a reusable canonical data model within the business process layer to simplify communication between layers.
+* Use messaging only where deemed necessary to support high-volume deployments.
+* Use inline sub-flows instead of top-level flows wherever possible.
+
+The assets being developed for the MuleSoft Accelerators were also intended to be flexible and extensible, supporting not only the use cases that have been solutioned but also other use cases needed by customers to support their business. Additional endpoints have been included in some APIs, especially System APIs shared across multiple use cases, which may not be required in all customer deployments. The reference architectures for most solutions have also been designed for long-term growth and evolution, which sometimes results in additional apps and interactions that are not always required for smaller deployments. 
+
+The following sections describe steps you can take to reduce the number of flows in accelerator apps.
+
+=== Remove Unused APIkit Handlers
+
+Remove scaffolded flows from the main router flow file in APIkit, which is usually the `*-api.xml` file. Leave the API specification unchanged instead of rescaffolding, because you always edit the file manually to remove the scaffolded flows. Avoid circumventing the APIkit router with a plain choice router, because that makes the router less discoverable and increases maintenance costs.
+
+Also, remove the flow for the APIkit console.
+
+=== Remove API Health Checks
+
+The `get:/ping` endpoint periodically monitors the health status of deployed APIs. Remove the handlers for these endpoints from the main APIkit flow file if monitoring is not needed. If you keep the endpoints, remove the checks for downstream dependencies to reduce messages and throughput.
+
+=== Generalize Event Sources
+
+Instead of creating monolithic flows that listen to generic events, combine event sources into more granular events. For example, there are general events for publishing and consuming updates to Parties, such as Individual and Household, and to Party Roles, such as Customer and Supplier. By combining the update queues, you bind them to both Exchanges and a single handler consumes them. You receive the same number of messages but with fewer flow event sources.
+
+Take the same approach for VMQueue subscribers, SFTP listeners, and others.
```

Comment:
"and others" makes me wonder which.

Suggest "Take the same approach for other <thingies> (events?), such as..."